Import config

axelchalon opened the pull request:
closes #1

axelchalon committed:
Import config

tomusdrw commented on src/components/Importer.js:
```diff
@@ -0,0 +1,68 @@
+        onClick={() => this.refs.fileInput.click()}>
+
+        <input
+          style={{display: 'none'}}
```
For performance reasons all callbacks and `style` should not be re-computed on every render.

axelchalon replied:
Cool! Didn't know about this. But because it's a PureComponent the render method gets called very rarely; and I read that doing `.bind(this)` also reallocates a new variable ; ended up using https://reactjs.org/docs/faq-functions.html#bind-in-constructor-es2015 although it might be overkill.

tomusdrw replied:
Actually a nice trick is to use lambdas for that, so instead of:
```
class X {
 constructor() {
   this.something = this.something.bind(this);
 }
 something () { console.log(this) }
}
```
you do
```
class X {
  something = () => console.log(this);
}
```
and `bind` is not needed.

axelchalon replied:
ah, nice! :+1:

axelchalon committed:
Grumbles (performance)

axelchalon committed:
Merge branch 'master' into import-config

axelchalon commented:
I'll update the generated data.json too; just a moment

axelchalon committed:
Fix list field; update account.password type

List field: previous code undefined the removed element in the
array, but didn't remove it; showed "undefined" as a result in
the Preview, etc.

tomusdrw commented:
Thanks!
